### PR TITLE
added control plane and not pool instance types

### PR DESCRIPTION
**Pull request by bkauf**
added support for changing instance types of cp and node pool from the variables file

### CODE REVIEW

File: anthos-multi-cloud/AWS/modules/kms/main.tf [hunk 1]
```diff
@@ -34,5 +34,5 @@ resource "aws_kms_key" "database_encryption_kms_key" {
 
 resource "aws_kms_alias" "database_encryption_kms_key_alias" {
   target_key_id = aws_kms_key.database_encryption_kms_key.arn
-  name          = "alias/${var.anthos_prefix}-database-encryption-key"
+  name          = "alias/-${var.anthos_prefix}-database-encryption-key"
```
Comment (hdp617) [inline anthos-multi-cloud/AWS/modules/kms/main.tf:37]: typo?

**Review by hdp617 — APPROVED** [review]

**Review by bkauf — COMMENTED** [review]
Removed and resubmitted

File: anthos-multi-cloud/AWS/variables.tf [hunk 2]
```diff
@@ -110,3 +110,13 @@ variable "node_pool_instance_type" {
   description = "AWS Node instance type"
   type        = string
 }
+
+variable "control_plane_instance_type" {
+  description = "AWS Node instance type"
+  type        = string
+}
+
+
```
[the remark this answers is not in the document]
Comment (bkauf) [reply anthos-multi-cloud/AWS/variables.tf:119]: Done

**Review by bkauf — COMMENTED** [review]
reviewed

**Review by bkauf — COMMENTED** [review]
reviewed

**Review by bkauf — COMMENTED** [review]
Added new line

**Review by bkauf — COMMENTED** [review]
removed extra blank lines

**Review by bkauf — COMMENTED** [review]
added new line

**Review by bkauf — COMMENTED** [review]
removed white space

**Pull request merged**